node/config.go

```diff
@@ -437,6 +437,7 @@ func makeAccountManager(conf *Config) (*accounts.Manager, string, error) {
 		// There is no datadir.
 		keydir, err = ioutil.TempDir("", "klaytn-keystore")
 		ephemeral = keydir
+		defer os.RemoveAll(keydir)
```

Comment:
Is it okay removing the dir here? Same as the comment I wrote below.

Reply:
Thank you the line is erased.